src/layer/vector/Polyline.js

```diff
@@ -172,6 +172,21 @@ export var Polyline = Path.extend({
 		return this.redraw();
 	},
 
+	// @method setLastLatLng(latlng: LatLng, latlngs? LatLng[]): this
+	// Modifies the last point of the polyline to be the specified point.
+	// By default, adds to the first ring of the polyline in case of a
+	// multi-polyline, but can be overridden by passing a specific ring as a
+	// LatLng array (that you can earlier access with
+	// [`getLatLngs`](#polyline-getlatlngs)).
+	setLastLatLng: function (latlng, latlngs) {
+		latlngs = latlngs || this._defaultShape();
+		latlng = toLatLng(latlng);
+		latlngs.pop();
+		latlngs.push(latlng);
+		this._bounds.extend(latlng);
```

Comment:
This will only extend the bounds, never make them smaller. This allows for an undesirable edge case where calling `setLatLngs()` with random points all over the globe and finally with a "good" point will make the polyline's bounds way larger than they should be.